### DISCUSSION TITLE
Search engines and incognito mode

### DISCUSSION THREAD

**Discussion by u2615** [bromite/bromite#1803] [Ideas]
Hey, been on this issue a whole day after installation so I thought maybe I'm not the only one and that could be of any use (maybe even added to the wiki?):

 In a scenario where one *starts* with setting privacy options like 'Open all tabs in incognito mode' (which is the first things I do on anything I install), and *then* tries to add search engines, it seems it can't be done (which in hindsight would be expected). I had to temporarily deactivate such options before I could add search engines. 

Edit: typos

**Comment by uazo** [comment]
> I add to temporarily deactivate such options before I could add search engines.

I confirm, I had already noticed, we were talking about it [here](https://github.com/bromite/bromite/issues/1615#issuecomment-986235226)
